src/commands.js

```diff
@@ -78,6 +78,60 @@ DeregisterHandler = async (ctx) => {
     await ctx.otherBot.telegram.sendMessage(telegramId, messages.DeregisterSuccess)
 }
 
+ConfirmHandler = async (ctx) => {
+    if (!ctx.isRegistered) {
+        return ctx.reply(messages.NotRegistered)
+    }
+    var person = ctx.person
```

Comment:
Is there a reason for this to be var instead of const? It might be even better to inline this